nucliadb/src/nucliadb/search/search/find.py

```diff
@@ -278,6 +224,14 @@ async def _external_index_retrieval(
         max_parallel_operations=50,
     )
 
+    # once hydrated, we send the results to rerank
+    reranker = query_parser.reranker
+    await reranker.rerank_find_response(
+        kbid, query_parser.query, retrieval_results, query_parser.page_size
+    )
+    # XXX: done later, could be substituted though
+    retrieval_results.best_matches.clear()
```

Comment:
Is this even needed? It's replaced 4 lines below.

Reply:
Removed :)